src/ucp/api/ucp.h

```diff
@@ -333,6 +335,19 @@ typedef enum {
 
 /**
  * @ingroup UCP_DATATYPE
+ * @brief Generate a reusable identifier for Scatter-gather IOV data type.
```

Comment:
1. how do you suggest we achieve this effect? how would IOV (currrent API - no handle) store the previously mapped memh?

2. I don't see how it will help, since "composite" requires reading from different regions of memory so need multiple pointers.